maestro-ai/README.md

```diff
@@ -0,0 +1,25 @@
+# maestro-ai
+
+This project implements AI support for use in Maestro.
+
+It's both a library and an executable demo app.
+
+### Demo app
+
+An API key is required. Set it with `MAESTRO_CLI_AI_KEY` env var. Examples:
+- OpenAI: `export MAESTRO_CLI_AI_KEY=sk-...`
```

Comment:
Is this Copilot API key, thats required?

Reply:
No, it can be any API key for OpenAI/Anthropic that works.

Reply:
Got it, thanks!